### PR TITLE
chore: Upload architecture diagrams

### PR DESCRIPTION
**Pull request by arbrown**
This PR also makes several changes necessary to get the CI to pass, including:
* Fixing Lint Errors
* Adjusting test timeouts
* Adjusting test environment setup

### CODE REVIEW

**Comment by arbrown** [comment]
This PR also changes root README.md to a symbolic link to infra/README.md

File: infra/README.md [hunk 1]
```diff
@@ -23,31 +23,29 @@ The resources/services/activations/deletions that this module will create/trigge
...
+| bucket\_location | Bucket location where the uploaded files will be stored. | `string` | `"US"` | no |
+| data\_init\_archive\_file\_name | Name of the archive file that contains the initialization data. | `string` | `"initialization.tar.gz"` | no |
+| data\_init\_bucket\_name | Name of the Cloud Storage bucket that store the archive file for initialization. | `string` | `"jss-resources"` | no |
+| labels | A map of key/value label pairs to assign to the cloud resources. | `map(string)` | <pre>{<br>  "app": "cloud-deployment-gce-java"<br>}</pre> | no |
 | project\_id | GCP project ID. | `string` | n/a | yes |
-| region | Google cloud region where the resource will be created. | `string` | `"us-central1"` | no |
-| source_image | Source VM image to deploy on GCE. | `string` | `"https://www.googleapis.com/compute/beta/projects/hsa-public/global/images/jss-cd-gce-vm-image"` | no |
-| zones | Gogle cloud zone where resources will be created. | `string` | `us-central1-a` | no |
+| region | GCP region where the cloud resource will be created. | `string` | `"us-central1"` | no |
```
Comment (bourgeoisor) [inline infra/README.md:34]: We're not supposed to use "GCP", the term has been deprecated in favour of "Google Cloud"

Comment (bourgeoisor) [reply infra/README.md:34]: Same comment elsewhere in this README.

**Review by bourgeoisor — CHANGES_REQUESTED** [review]

File: infra/test/integration/simple_example/simple_example_test.go [hunk 2]
```diff
@@ -63,7 +63,7 @@ func TestSimpleExample(t *testing.T) {
 			}
 			return false, nil
 		}
-		utils.Poll(t, isBlueMigServing, 20, time.Second*20)
+		utils.Poll(t, isBlueMigServing, 10, time.Minute*2)
```
Comment (bourgeoisor) [inline infra/test/integration/simple_example/simple_example_test.go:66]: These changes don't seem related to the architecture diagram / README :)

What does this diff do?

Comment (arbrown) [reply infra/test/integration/simple_example/simple_example_test.go:66]: I was troubleshooting the failing CFT tests thinking it was related to timeouts, but it appears not to be.  I'll roll back.

**Review by bourgeoisor — APPROVED** [review]
LGTM!
